### DISCUSSION TITLE
Hide parent element if all children are hidden

### DISCUSSION THREAD

**Discussion by pepew-le-boss** [tailwindlabs/tailwindcss#12803] [Help] [answered]
I have a parent `<div>` that I want to hide when it's children are all hidden.

I've tried something like `has-[div:hidden]:hidden` but it's not working.

```javascript
<div class="has-[div:hidden]:hidden flex flex-col gap-5">
    <div class="hidden md:flex">
        some content
    </div>
</div>
```

**Comment by wongjn** [comment, marked as answer]
You could do `hidden has-[:not(.hidden)]:flex` which applies `display: flex` if at least one child does not have the `hidden` class as per [this Tailwind Play](https://play.tailwindcss.com/I9hfG0HWLV) but be aware that this does not "respond" in anyway to the `md:flex`.
